scripts/score_tractogram.py

```diff
@@ -49,35 +48,29 @@ def buildArgsParser():
     p = argparse.ArgumentParser(description=DESCRIPTION,
                                 formatter_class=argparse.RawTextHelpFormatter)
 
-    p.add_argument('tractogram', action='store',
-                   metavar='TRACTS', type=str, help='Tractogram file')
-
-    p.add_argument('base_dir', action='store',
-                   metavar='BASE_DIR', type=str,
+    p.add_argument('tractogram', metavar='TRACTS', type=str,
```

Comment:
You removed the `action` since it's the default value. You can do the same with the `type` when it's a string.